src/pymovements/plotting/traceplot.py

```diff
@@ -37,44 +42,59 @@
 if 'pytest' in sys.modules:  # pragma: no cover
     matplotlib.use('Agg')
 
-DEFAULT_SEGMENTDATA = {
+LinearSegmentedColormapType: TypeAlias = Dict[
```

Comment:
can you use `dict` instead of `Dict` here?

Reply:
no unfortunately only works for 3.9+ and we (still) support 3.8

Reply:
are you sure? `from __future__ import annotations` should fix that.

we're using this kind of type annotations everywhere else in the package, for example here:

https://github.com/aeye-lab/pymovements/blob/5e0959fb9ed209850733645eab5fc8ae77d31c3f/src/pymovements/synthetic/step_function.py#L28-L34

Reply:
I couldn't resist and had to test it by myself and you're right, it doesn't work.

I just absolutely don't understand why that's the case, because the future annotations solve this issue everywhere else (even in the same file there is `tuple[int, int]`.